scripts/vendor-types.sh

```
@@ -0,0 +1,54 @@
+echo "Vendoring proto2type domain types from: ${GEN_DIR}"
+echo ""
+
+copy() {
+  local src="${GEN_DIR}/$1"
+  local dst="$2"
+  if [ ! -f "${src}" ]; then
+    echo "  SKIP  $1 (not found)"
+    return
+  fi
+  cp "${src}" "${dst}"
+  echo "  COPY  $1 → ${dst#${REPO_DIR}/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)